govcd/catalog.go

```diff
@@ -55,3 +76,352 @@ func (c *Catalog) FindCatalogItem(catalogitem string) (CatalogItem, error) {
 
 	return CatalogItem{}, fmt.Errorf("can't find catalog item: %s", catalogitem)
 }
+
+/**
```

Comment:
Please convert to // style of comments to match existing code.  Check Go system packages for examples--we want to match their style both in terms of syntax as well as the content of documentation. 

More generally the comment here should describe the user-visible interface specification.  In other words--what does this function accomplish for the client, what outputs/errors does it generate?  Please push implementation details (which are very helpful) into the implementation code or routines it calls.

Reply:
Fixed